spec/rubocop/cop/rspec/factory_bot/create_list_spec.rb

```diff
@@ -197,55 +208,55 @@
     it 'flags usage of create_list' do
       expect_offense(<<~RUBY)
         create_list :user, 3
-        ^^^^^^^^^^^ Prefer 3.times.
+        ^^^^^^^^^^^ Prefer 3.times.map.
       RUBY
 
       expect_correction(<<~RUBY)
-        3.times { create :user }
+        3.times.map { create :user }
       RUBY
     end
 
     it 'flags usage of create_list with argument' do
       expect_offense(<<~RUBY)
         create_list(:user, 3, :trait)
-        ^^^^^^^^^^^ Prefer 3.times.
+        ^^^^^^^^^^^ Prefer 3.times.map.
       RUBY
 
       expect_correction(<<~RUBY)
-        3.times { create(:user, :trait) }
+        3.times.map { create(:user, :trait) }
       RUBY
     end
 
     it 'flags usage of create_list with keyword arguments' do
       expect_offense(<<~RUBY)
         create_list :user, 3, :trait, key: val
-        ^^^^^^^^^^^ Prefer 3.times.
+        ^^^^^^^^^^^ Prefer 3.times.map.
       RUBY
 
       expect_correction(<<~RUBY)
-        3.times { create :user, :trait, key: val }
+        3.times.map { create :user, :trait, key: val }
       RUBY
     end
 
     it 'flags usage of FactoryGirl.create_list' do
       expect_offense(<<~RUBY)
         FactoryGirl.create_list :user, 3
-                    ^^^^^^^^^^^ Prefer 3.times.
+                    ^^^^^^^^^^^ Prefer 3.times.map.
       RUBY
 
       expect_correction(<<~RUBY)
-        3.times { FactoryGirl.create :user }
+        3.times.map { FactoryGirl.create :user }
       RUBY
     end
 
     it 'flags usage of FactoryGirl.create_list with a block' do
       expect_offense(<<~RUBY)
         FactoryGirl.create_list(:user, 3) { |user| user.points = rand(1000) }
-                    ^^^^^^^^^^^ Prefer 3.times.
+                    ^^^^^^^^^^^ Prefer 3.times.map.
       RUBY
 
       expect_correction(<<~RUBY)
-        3.times { FactoryGirl.create(:user) } { |user| user.points = rand(1000) }
+        3.times.map { FactoryGirl.create(:user) } { |user| user.points = rand(1000) }
```

Comment:
Is this even valid? Perhaps you've uncovered a bug in the original autocorrection :sweat_smile:

Reply:
Oh, thanks, I hadn't noticed that until you pointed it out. I created another Pull Request for this 👍 

- https://github.com/rubocop/rubocop-rspec/pull/1472